perf-test/README.md

```diff
@@ -0,0 +1,9 @@
+### Test run command
+
+```
+mvn test -Ddatabase.connectionString="jdbc:mysql://localhost:3306/demo" \
```

Comment:
user should decide what test to run explicitly defining test name

Reply:
done